### PR TITLE
added rotation example to example gallery

### PR DESCRIPTION
**Pull request by kolibril13**
Added a MovingAngle example to the example gallery which showcases an animated scene of the new Angle class.

### CODE REVIEW

File: docs/source/examples.rst [hunk 1]
```diff
@@ -214,6 +214,55 @@ Animations
             self.play(square.animate.scale(0.3))
             self.play(square.animate.rotate(0.4))
 
+.. manim:: MovingAngle
+    :ref_modules: manim.mobject.geometry.Angle
+    :ref_functions: Mobject.rotate
```
Comment (kolibril13) [inline docs/source/examples.rst:219]: This reference seems to be broken.
Not only in this pr, but in all reference fields in the example gallery

**Comment by kolibril13** [comment]
Rendered example here :
https://manimce--1204.org.readthedocs.build/en/1204/examples.html#movingangle

File: docs/source/examples.rst (hunk 1 above)
Comment (behackl) [reply docs/source/examples.rst:219]: If the referenced module, class, or function doesn't have a docstring, then there is nothing to link to, making the link appear broken. `Mobject.rotate` doesn't have a docstring.

Also, `manim.mobject.geometry.Angle` isn't a module -- you want to reference the class in this case.

**Review by jsonvillanueva — APPROVED** [review]
Example looks good, approving with/without the docstring in Mobject.rotate

**Comment by kolibril13** [comment]
> Example looks good, approving with/without the docstring in Mobject.rotate

Nice. I will write the docstrings in a separate pr.

**Comment by jsonvillanueva** [comment]
Yes, I think a separate PR would be better but don't mind doing it here as it's quite a small documentation/addition. I'll let @behackl merge/approve though. I also think this example can be revisited once `Angle` is able to rotate more than 360 degrees

**Pull request merged**